test/nodes/test_eval_cobalt.py

```
@@ -0,0 +1,500 @@
+import copy
+import sys
+from types import ModuleType
+from unittest.mock import MagicMock, patch
+
+
+# ---------------------------------------------------------------------------
+# Mock infrastructure — rocketlib, ai.common, depends, cobalt
+# ---------------------------------------------------------------------------
+
+
+def _setup_mocks():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_setup_mocks`

(ANN202)